Rules.md

```diff
@@ -203,15 +203,16 @@ A battle has one or more rounds. Every round has following parts:
 1. **Attacker decides to retreat or has no healthy units left:** All units move back to the last
    touched unoccupied, sea or owned region.
 1. **Defender decides to retreat or has no healthy units left:** All units (including wounded units)
-   move to an adjacent land region that is unoccupied or owned by the defender.
+   move to a single adjacent land region that is unoccupied or owned by the defender. Exception:
+   units can't retreat to the region where the attacker's move token was resolved.
 
   If no such regions are found, the units are killed. If there is choice, the defender may choose
   the region.
 
   Clarifications for edge cases:
 
-    * If the attacker completely **vacated a region** to perform this attack, it could happen that
-      the defending units retreat to this region.
+    * If the attacker completely **vacated a region** to attack an adjacent region, defending units
```

Comment:
as with the other comment you already clearly stated retreat is towards an adjacent land region, so yes it would be redundant (doesn't change the fact that I don't mind this specific one being kept)